app/models/review.rb

```diff
@@ -0,0 +1,9 @@
+class Review < ActiveRecord::Base
+  validates :rating, presence: true, numericality: { only_integer: true, greater_than: 0, less_than: 6 }
```

Comment:
separate these by commas like this: 
```ruby
validates :rating, 
  presence: true, 
  numericality: { only_integer: true, greater_than: 0, less_than: 6 }
```
Also assocations belong above validations. So belongs_to :user should be at the top